docker-compose.yml

```diff
@@ -3,33 +3,60 @@
 version: '3.1'
 
 services:
+  calculate-journey-variable-payments:
+    restart: on-failure:3
```

Comment:
This is a workaround that needs addressing.  The OAuth2 service takes too long to start so this service fails to start initially.